subworkflows/local/bwa-pre-process.nf

```diff
@@ -67,7 +72,7 @@ workflow BWA_PREPROCESS {
     QC_REPORT(ch_qcreport_input, reference[0])
 
     ch_versions            = ch_versions.mix(  SEQKIT_PAIR.out.versions, 
-                                               SEQTK_SAMPLE.out.versions, 
+                                               SEQTK_SAMPLE.out.versions 
```

Comment:
This line might throw an error if SEQTK_SAMPLE doesnt run, but need to validate with testing.

Reply:
You are totally right. I was running my tests on a single sample. So it was failing when trying to make the trees but the other results looked good. I think thats how I missed it. I am going to remove this line. Thanks.